addons/l10n_fi/data/account_tax_template_data.xml

```diff
@@ -20,7 +20,7 @@
               'factor_percent': 100,
               'repartition_type': 'tax',
               'account_id': ref('account_2930'),
-              'plus_report_line_ids':  [ref('tax_report_sales_25_5')],
+              'plus_report_line_ids':  [ref('tax_report_sales_24')],
```

Comment:
I don't think these should change

Reply:
25.5% taxes should never have been put in a separate line in the first place regarding the documentation and the partner's feedback so I stil believe the 25.5% should have been put in the 24% report line.

Reply:
> Using MyTax for VAT reporting
When the VAT taxpayer’s tax period is the month, the VAT return in MyTax will display the following space starting 1 September 2024: “VAT on domestic sales by tax rate: 25.5% VAT”.

IMO, the 25.5% tax tag should stay